Formula/go.rb

```diff
@@ -1,15 +1,16 @@
 class Go < Formula
   desc "Open source programming language to build simple/reliable/efficient software"
   homepage "https://go.dev/"
-  url "https://go.dev/dl/go1.20.6.src.tar.gz"
-  mirror "https://fossies.org/linux/misc/go1.20.6.src.tar.gz"
-  sha256 "62ee5bc6fb55b8bae8f705e0cb8df86d6453626b4ecf93279e2867092e0b7f70"
+  url "https://go.dev/dl/go1.21rc3.src.tar.gz"
+  mirror "https://fossies.org/linux/misc/go1.21rc3.src.tar.gz"
+  sha256 "9de96a2f15196a9c69cb9ab122c9c2aaecca30ec9448caa71543eaeb19ca91ab"
   license "BSD-3-Clause"
   head "https://go.googlesource.com/go.git", branch: "master"
 
   livecheck do
-    url "https://go.dev/dl/"
-    regex(/href=.*?go[._-]?v?(\d+(?:\.\d+)+)[._-]src\.t/i)
+    skip "testing rc2"
+    # url "https://go.dev/dl/"
+    # regex(/href=.*?go[._-]?v?(\d+(?:\.\d+)+)[._-]src\.t/i)
```

Comment:
To be reverted back to:
```suggestion
    url "https://go.dev/dl/"
    regex(/href=.*?go[._-]?v?(\d+(?:\.\d+)+)[._-]src\.t/i)
```